docs/content/guides/developer/getting-started.mdx

```diff
@@ -31,11 +31,20 @@ Go to [Query Sui RPC with GraphQL](./getting-started/graphql-rpc.mdx).
 
 ## Move IDEs and plugins
 
-The recommended IDE for Move development is Visual Studio Code with the [Move extension](https://marketplace.visualstudio.com/items?itemName=mysten.move) available in the Visual Studio Marketplace. Follow the Visual Studio Marketplace instructions to install the extension.
+The following table lists proven choices for IDEs with Move extensions. These IDEs and plugins are not required, but they provide a better experience for developing on Sui.
 
-See more [IDE options](https://github.com/MystenLabs/awesome-move#ides) in the [Awesome Move](https://github.com/MystenLabs/awesome-move) documentation.
+| IDE | Move plugin | Description |
+| --- | ------ | ----------- |
+| [Visual Studio Code](https://code.visualstudio.com/) | &bull; [Move extension](https://marketplace.visualstudio.com/items?itemName=mysten.move)<br/>&bull; [Move syntax](https://marketplace.visualstudio.com/items?itemName=damirka.move-syntax) | The recommended free IDE for Move development from Microsoft. |
+| [IntelliJ IDEA](https://www.jetbrains.com/idea/) | [Move Language Plugin](https://plugins.jetbrains.com/plugin/14721-move-language) | A commercial IDE from JetBrains. |
+| [Emacs](https://www.gnu.org/software/emacs/) | [move-mode](https://github.com/amnn/move-mode) | A free and open source text editor. |
```

Comment:
Psst @tzakian -- you should get (Neo)vim in here (Now you can't say I didn't make this a fair fight 😉)

Reply:
(Same probably applies to the Move Book)